packages/compass-components/src/index.ts

```diff
@@ -183,3 +183,4 @@ export { usePersistedState } from './hooks/use-persisted-state';
 export { GuideCue, GuideCueProvider } from './components/guide-cue/guide-cue';
 export type { Cue, GroupCue } from './components/guide-cue/guide-cue';
 export { PerformanceSignals } from './components/signals';
+export { ToastBody } from './components/toast-body';
```

Comment:
Is this generic enough for compass-components? Are we planning to update some other places, like import-export, to use that component instead?

Reply:
They are already updated in that other places.

And it's already in two different places, so it makes sense to extract it to components. We might need later more toasts that have an action.

Reply:
Ah, I see, github collapsed filenames in a weird way. If we need more toasts later, we should probably update the interface in a way that doesn't require us to import ToastBody separate from openToast method, but that's good enough for now